src/main/java/io/kestra/plugin/googleworkspace/drive/DriveService.java

```diff
@@ -0,0 +1,34 @@
+package io.kestra.plugin.googleworkspace.drive;
+
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.json.gson.GsonFactory;
+import com.google.api.services.drive.Drive;
+import com.google.auth.http.HttpCredentialsAdapter;
+import com.google.auth.oauth2.GoogleCredentials;
+import io.kestra.core.runners.RunContext;
+
+import java.io.ByteArrayInputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Collections;
+
+public class DriveService {
```

Comment:
Please remove it and create a `AbstractDriveTrigger` extending `AbstractTrigger` where you would put the same connection logic (like it's done [here](https://github.com/kestra-io/plugin-googleworkspace/blob/master/src/main/java/io/kestra/plugin/googleworkspace/drive/AbstractDrive.java))